src/com/trilead/ssh2/packets/PacketKexDHReply.java

```diff
@@ -15,7 +13,7 @@
 	byte[] payload;
 
 	byte[] hostKey;
-	BigInteger f;
+	byte[] publicKey;
```

Comment:
Binary incompatibility, use a new version of this call is you can't keep the fields the same (and hide all fields in the new version)